### ISSUE TITLE
Why so many training iterations ? (500 + 200) * 4000

### ISSUE THREAD

**Issue by Honlan** [tengteng95/Pose-Transfer#49]
Thanks a lot for this awesome paper as well as the open-source codes.

In the paper, the authors claim that the model is trained with Adam for 90K iterations, where the learning rate decays to 0 during the latter 60K iterations.

However, in the codes I find that the total iteration is set to (500 + 200) * 4000, which is far more than the setting of the paper.

Could you please explain this for me? Thanks again.

**Comment by jessemelpolio** [comment]
Actually, the number of iterations depends on the batch size. We set 32 as our batch size, meaning 1 iteration contains 32 images. So the total number of iterations is roughly equal to 90K.

**Comment by dawn7564** [comment]
Sorry, I'm still not clear to that.  

1 epoch contains 4000 iters when batchsize is 32(just like the output when training "epoch: ***, iters: 4000, time: 0.070" ), and there are 500 + 200 = 700 epoch, so I think there are 700 * 4000 = 2800K iters which mean that the weights update 2800K times not 90K.

I know I made a mistake somewhere, can you please tell me where I am wrong? 

thank you :)  @jessemelpolio

**Comment by tengteng95** [comment]
Hi, there are 700*4000=2800K "samples" used for training and each iteration contains 32 "samples". Hence, the iterations are 2800K/32, which is around 90K. For simplicity, in our code, we directly output "iters 4000" instead of "iters 4000/32". Sorry for bringing the misunderstandings.

**Comment by Honlan** [comment]
got it. thanks a lot
